mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketSubscriber.java

```diff
@@ -31,7 +31,10 @@ public void subscribe(MqttMessageListener listener) {
             return;
         }
 
-        final String token = tokenSupplier.get();
+        final String token = tokenSupplier
+                .get()
+                .getTokenString();
```

Comment:
I would avoid method chain and just expose `getTokenString` and remove `get` completely like @gapa-c8y suggest.

Reply:
unsubscribe needs a Token. 
I am returning the Token to the caller. I think if required, getTokenString can be a separate method but I don't think that is required. 
Gary's point (if I haven't misunderstood) was just around formatting.

Reply:
Yeah I was referring to formatting. But I was hovering around similar thoughts made by @slc-softwareag  in other comment - it's not a token-supplier

Reply:
What I mean is that currently `TokenSupplier` becomes more like a `TokenService` where we have two public methods: `getTokenString`, `unsubscribe`. We don't have to expose `Token get()` as in such case it will be used just internally

Reply:
I would rather prefer to do get().getTokenString() from the caller as I think that is neat.

Reply:
I have committed the changes. Let me if there is a strong objection to get() returning Token obj.